samples/nhibernate/simple/NHibernate_6/Client/App.config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="UnicastBusConfig" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core" />
+  </configSections>
+
+  <connectionStrings>
+    <add name="NServiceBus/Persistence" connectionString="Data Source=.;Database=Samples.NHibernate;Integrated Security=True" providerName="System.Data.SqlClient" />
```

Comment:
the standard for samples is a names sql instance. eg http://docs.particular.net/samples/saga/nh-custom-sagafinder/#code-walk-through-nhibernate-setup